src/coreComponents/fileIO/Outputs/BlueprintOutput.cpp

```diff
@@ -38,17 +38,54 @@ namespace internal
  * @brief @return The Blueprint shape from the GEOSX element type string.
  * @param elementType the elementType to look up.
  */
-string toBlueprintShape( string const & elementType )
+string toBlueprintShape( ElementType const elementType )
 {
-  static std::unordered_map< string, string > const map =
+  switch( elementType )
   {
-    { "C3D8", "hex" },
-    { "C3D4", "tet" }
-  };
+    case ElementType::Tetrahedon:
+    { return "tet";}
+    case ElementType::Hexahedron:
+    { return "hex";}
+    default:
+    {
+      GEOSX_ERROR( "No Blueprint type for element type: " << elementType );
+      return {};
+    }
+  }
+}
 
-  auto const iter = map.find( elementType );
-  GEOSX_ERROR_IF( iter == map.end(), "No Blueprint type for " << elementType );
-  return iter->second;
+static std::vector< int > getBlueprintNodeOrdering( ElementType const elementType )
+{
+  // Same as VTK, but kept separate for flexibility
+  switch( elementType )
+  {
+    case ElementType::Line:
+    { return { 0, 1 };}
+    case ElementType::Triangle:
+    { return { 0, 1, 2 };}
+    case ElementType::Quadrilateral:
+    { return { 0, 1, 2, 3 };                                // TODO check
+    }
+    case ElementType::Polygon:
+    { return { 0, 1, 2, 3, 4, 5, 6, 7, 8 };                          // TODO
+    }
+    case ElementType::Tetrahedon:
+    { return { 1, 0, 2, 3 };}
+    case ElementType::Pyramid:
+    { return { 0, 3, 2, 1, 4, 0, 0, 0 };}
+    case ElementType::Prism:
+    { return { 0, 4, 2, 1, 5, 3, 0, 0 };}
+    case ElementType::Hexahedron:
+    { return { 0, 1, 3, 2, 4, 5, 7, 6 };}
+    case ElementType::Polyhedron:
+    { return { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 };                             // TODO
+    }
```

Comment:
Uncrustify is absolutely ruining it.. objections to allowing un-braced case statements? E.g.
```c++
case A: return 1;
case B: return 2;
```

Reply:
I think I was trying to avoid unbraced multi-line case statements. I don't think there is any sort of real issue with relaxing the formatting for this...I recall wanting to be consistent with always having if-statments braced. I have no objection to what you have done.